mission/landmarks/src/landmarks.cpp

```diff
@@ -0,0 +1,32 @@
+#include "landmarks.h"
+
+Landmarks::Landmarks ():loop_rate(10) {
+    op_sub = n.subscribe("object_positions_in",10, &Landmarks::callback, this);
```

Comment:
should i add some namespace to the topic names?

Reply:
I don't foresee that the landmarks file will grow massively, so it's not *necessary*, but it doesn't hurt to have either.